File: Networking/Response/GetParticipantsForEventWithCountResponse.cs
using System.Text.Json.Serialization;
using mpp_proiect_csharp_DianaGliga11.Model.DTO;

namespace Networking.Response
{
    
    public class GetParticipantsForEventWithCountResponse: IResponse
    {
        [JsonPropertyName("participants")]
        public List<ParticipantDTO> participants{ get; set; }

        [JsonConstructor]
        public GetParticipantsForEventWithCountResponse()
        {
            
        }

        public GetParticipantsForEventWithCountResponse(List<ParticipantDTO> participants)
        {
            this.participants = participants;
        }
    }
}

File: Networking/src/main/java/response/GetParticipantsForEventWithCountResponse.java
package response;

import DTO.ParticipantDTO;

import java.util.Collection;

public class GetParticipantsForEventWithCountResponse implements Response{
    private final Collection<ParticipantDTO> participants;

    public GetParticipantsForEventWithCountResponse(Collection<ParticipantDTO> participants) {
        this.participants = participants;
    }

    public Collection<ParticipantDTO> getParticipants() {
        return participants;
    }
}
